### ISSUE TITLE
[Bug] Repair not working on Windows 11

### ISSUE THREAD

**Issue by maicol07** [BetterDiscord/Installer#243]
**Describe the Bug**
Repair not working on Windows 11 since the `wmic` command has been deprecated in Windows 10 and then removed in Windows 11 and should be replaced with WMI: https://docs.microsoft.com/en-us/powershell/scripting/learn/ps101/07-working-with-wmi?view=powershell-7.2

**To Reproduce**
1. Open BD installer
2. Select Repair when choosing the action
3. See error

**Expected Behavior**
Repair works

**Screenshots**
```
Starting Repair...

Killing Discord...
Attempting to kill DiscordCanary
❌ Could not kill Discord Canary
❌ Command 'WMIC path win32_process get Name,Processid,ParentProcessId,Commandline,ExecutablePath' terminated with code: 1

The repair seems to have failed. If this problem is recurring, join our discord community for support. https://discord.gg/0Tmfo5ZbORCRqbAd
```

**Discord Version**
Canary

**Comment by SmolAlli** [comment]
Try using "install" instead. It does essentially the same thing.

**Comment by maicol07** [comment]
Yes, it works. However, the last step (opening Discord) fails because it runs a `wmic` command

**Comment by zerebos** [comment]
This is an issue with the installer, moving to the correct repo...

**Comment by tropicaaal** [comment]
Closing as dupe of #232, fixed by #235.
